path_issue_sign.go

```diff
@@ -1,240 +1,228 @@
 package keyfactor
 
-import (
```

Comment:
I think `path_certs.go` now implements the functionality this provided. Should we just delete this file?